code-challenges.test.js

```diff
@@ -14,38 +14,139 @@
+//   })
+// })
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
 
+
+// ● coolPeeps › takes in an array of objects and returns an array with a sentence about each person with their name capitalized
+
+// ReferenceError: coolPeeps is not defined
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+// Snapshots:   0 total
+// Time:        0.174 s
 // b) Create the function that makes the test pass.
 
+//use filter to get an array of first and last name
+// ****PSEUDO CODE
+// create a function 
+// input is array of strings
+// need to create a method that will give me just the strings...split(?)
+// use another function to capitalize the first letters 
+// need to join the strings for name/occupation output
+
+
+
+
+// const coolPeeps  = (array) => {
+//   return array.map((value) => {
+//     let splitArr = value.name.split(' ')
+//     let name = splitArr[0][0].toUpperCase() + splitArr[0].substring(1) + " "
+//     + splitArr[1][0].toUpperCase() + splitArr[1].substring(1)
+//     return `${name} is ${value.occupation}`
+//   })
+// }
 
 // --------------------2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
 // Expected output: [ 2, 0, -1, 0 ]
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+
 // Expected output: [ 2, 1, -1 ]
 
+// describe("daWhat", () => {
+//   it("takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3", () => {
+//    const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+//    const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+//       expect(daWhat(hodgepodge1)).toEqual([2, 0, -1, 0])
+//       expect(daWhat(hodgepodge2)).toEqual([2, 1, -1])
+//    })
+// })
 
-// b) Create the function that makes the test pass.
+// ● daWhat › takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3
 
+// ReferenceError: daWhat is not defined
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+// Snapshots:   0 total
+// Time:        0.15 s
+// // // // b) Create the function that makes the test pass.
+
+// ****PSEUDO CODE
+// create a function with array as param
+// input is array of numbers and strings
+// need to create a method that will filter out the numbers 
+// return an array of numbers
+// return using .map to go thru the array and divide by 3 to get remaining numbers
+
+// 
+//   daWhat = (array) => {
+//    let numArry = array.filter(value => {
+//      return typeof value === 'number'
+//     })
+//   return  numArry.map(value => value % 3)
+//     }
```

Comment:
Logic looks excellent!  Make sure to use the javascript keyword const to declare the function on line 94.